src/ui/camera.js

```diff
@@ -162,7 +162,7 @@ class Camera extends Evented {
      * Returns the map's geographical centerpoint.
      *
      * @memberof Map#
-     * @returns The map's geographical centerpoint.
+     * @returns {LngLatLike} The map's geographical centerpoint.
```

Comment:
```suggestion
     * @returns {LngLat} The map's geographical centerpoint.
```
[LngLatLike](https://docs.mapbox.com/mapbox-gl-js/api/geography/#lnglatlike) is a flexible object for inputs. I presume the return value would be a LngLat.

Reply:
Noticing that we also have: `@property {LngLatLike} center The desired center.` So maybe I'm wrong about this.

Reply:
I see the example below says `return a LngLat object such as {lng: 0, lat: 0}` 
so I think you are correct here! @SnailBones